Draft.md

```diff
@@ -156,235 +156,171 @@ Eine Personengruppe, die besonders oft zum passiven Spielball von Forschungssoft
+- Kurzbeschreibung: ...
+- Rezensierte Version: ...
+- Plattform: ...
+- URLs und Entwicklerkontakt: ...
+- Publikationen: ...
+- Lizenzierung und Codeverfügbarkeit: ...
+- Alternativprodukte: ...
+
+Als Kontext verstehen wir eine Einordnung in das archäologische Forschungsfeld und Angaben zum möglichen Zusammenhang mit Forschungsprojekten, Arbeitsgruppen oder Institutionen. Für die kritische Beurteilung aus unterschiedlicher Perspektive haben wir unten einen Katalog an Fragen zusammengestellt. Zuletzt sollten die Ergebnisse der Betrachtung in einer Stellungnahme zusammengefasst werden, die die Software hinsichtlich ihrer Nützlichkeit, ihrer Bedienbarkeit, ihrer handwerklichen Qualität und ihrer Position in Relation zu den Idealen guter Forschungssoftware (z.B. FAIR und CARE), beurteilt.
+
+Selbstverständlich setzt die Rezension einer Forschungssoftware die Sichtung der Dokumentation und begleitender Publikationen ebenso voraus, wie die praktische Erprobung der Software selbst. Dabei ist eine realistische und transparente Einschätzung der eigenen Kompetenzen und des eigenen Nutzungsinteresses ein wichtiger Anhaltspunkt für die Leser, denn die Leserschaft ist wesentlich heterogener hinsichtlich ihrer Interessen, als man es bei der Besprechung einer Fachpublikation annehmen darf. Es ist eine wichtige Information, ob die Rezension rein aus Anwenderperspektive geschrieben wurde, oder auch eine Entwicklerperspektive umfasst.  
 
 ## Fragenkatalog zur Beurteilung von Software
 
 Im folgenden stellen wir einen kommentierten Fragenkatalog vor, der auf Kriterien zur Beurteilung von archäologischer Forschungssoftware hinarbeitet. Er ist untergliedert in drei Bereiche, die jeweils Fragen aus verschiedenen Kompetenzbereichen bündeln. Dabei bilden die ersten zwei Bereiche die wissenschaftliche Verwendung und die Anwendung und Bedienbarkeit aus Nutzersicht ab. Der dritte Bereich enthält Fragen, die für Entwickler und IT-Administratoren von hohem Interesse sind und die daher die Anwendbarkeit und Nachhaltigkeit in den Blick nehmen. Im Anschluss stellen wir Merkmale vor, die eventuell tabellarisch einer Rezension beigegeben werden sollten, aber nur im Ausnahmefall vom Rezensenten kritisch beurteilt werden.
-Wie bei der Besprechung einer wissenschaftlichen Publikation ist die Zusammensetzung und Gewichtung der einzelnen Merkmale vom Rezensenten selbst zu bestimmen und in Relation zum Thema zu setzen. Entsprechend verstehen wir unseren kommentierten Fragenkatalog auch als eine Art Maximalversion, die bei der Anfertigung und der Einschätzung der eigenen Kompetenzen Hilfe geben kann.
+
+Wie bei der Besprechung einer wissenschaftlichen Publikation ist die Zusammensetzung und Gewichtung der einzelnen Merkmale von der Rezensentin selbst zu bestimmen und in Relation zum Thema zu setzen. Entsprechend verstehen wir unseren kommentierten Fragenkatalog auch als eine Art Maximalversion, die bei der Anfertigung und der Einschätzung der eigenen Kompetenzen Hilfe geben kann.
 
```

Comment:
Die Archäologischen Informationen verwenden durchgehend die männliche Form, die dann für alle Geschlechter steht. Die Wahl der männlichen Form in unserem Draft ist somit von außen vorgegeben.

Reply:
Ok - Verzeihung. Das war mir nicht klar.

Reply:
Es ist auch eine Regelung, die mir so wenig gefällt, dass es mir widerstrebt auf die Umsetzung hinzuweisen :§